recipe/meta.yaml

```
@@ -12,28 +12,30 @@ source:
 
 build:
   skip: true  # [python_impl == 'pypy']
-  script: |
-    {{ PYTHON }} -m pip install . -vv
+  script:
+    - set CMAKE_GENERATOR=Ninja # [win]
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
-  script_env:   # [win]
-    # Unset `CMAKE_GENERATOR` and let Visual Studio set to its default
-    - CMAKE_GENERATOR=  # [win]
+  missing_dso_whitelist:  # [s390x]
+    - $RPATH/ld64.so.1    # [s390x]
 
 requirements:
   build:
     - python                              # [build_platform != target_platform]
     - cross-python_{{ target_platform }}  # [build_platform != target_platform]
     - {{ compiler('cxx') }}
-    - {{ stdlib("c") }}
     - cmake >=3.15  # to support `CMAKE_GENERATOR`
     - make                                # [not win]
+    - ninja                               # [win]
   host:
     - pip
     - python
-    - pybind11 >=2.11.1
+    - setuptools
+    - wheel
+    - pybind11 2
```

```suggestion
    - pybind11 2
```
Is this basically like `pybind11 2*`?

---

yes!